test/e2e/util.go

```diff
@@ -2733,3 +2734,36 @@ func getPodLogsInternal(c *client.Client, namespace, podName, containerName stri
 	}
 	return string(logs), err
 }
+
+func EnsureResourcesDeleted(ip, portRange string) error {
+	if testContext.Provider == "gce" || testContext.Provider == "gke" {
+		return ensureGCEResourcesDeleted(ip, portRange)
+	}
+	return nil
+}
+
+func ensureGCEResourcesDeleted(ip, portRange string) error {
```

Comment:
will we add other checks to this? otherwise, the name seems too general for only checking forwarding rules.